### ISSUE TITLE
Triggered even without using `watch` command

### ISSUE THREAD

**Issue by reubano** [Arcesilas/brunch-browser-sync#2]
```bash
reubano@tokpro [polythene-mithril-static] ⚡ brunch watch
[Browsersync] Access URLs:
 ----------------------------------
       Local: http://localhost:3000
    External: http://10.0.0.14:3000
 ----------------------------------
          UI: http://localhost:3001
 UI External: http://localhost:3001
 ----------------------------------
[Browsersync] Serving files from: public/
[Browsersync] Watching files...

reubano@tokpro [polythene-mithril-static] ⚡ brunch build
[Browsersync] Access URLs:
 ----------------------------------
       Local: http://localhost:3000
    External: http://10.0.0.14:3000
 ----------------------------------
          UI: http://localhost:3001
 UI External: http://localhost:3001
 ----------------------------------
[Browsersync] Serving files from: public/
[Browsersync] Watching files...
```

**Comment by Arcesilas** [comment]
Thanks for reporting this issue.
This happens because the configuration is the same, whether Brunch is building or watching. I need to find out how to know what Brunch is doing...

**Comment by Arcesilas** [comment]
Fixed in v0.1.2. Please update to latest version.
